lib/admin/components/AdminPage.js

```diff
@@ -0,0 +1,181 @@
+          <Button
+            bsStyle='danger'
+            bsSize='large'
+            href='https://manage.auth0.com/#/logs'>
+            <Icon type='star' /> View application logs on Auth0.com
+          </Button>
+        </p>
+      case 'servers':
+        if (!isApplicationAdmin || !isModuleEnabled('deployment')) return restricted
+        return <ServerSettings editDisabled={false} />
+      default:
+        return null
+    }
+  }
+
+  _getLinks = (isApplicationAdmin: boolean): React.Node[] => {
+    const links = [{ to: '/admin/users', children: <LGI>User management</LGI> }]
+    // Do not show non-appAdmin users these application-level settings
+    if (isApplicationAdmin) {
+      if (!isModuleEnabled('enterprise')) {
+        links.push({ to: '/admin/organizations', children: <LGI>Organizations</LGI> })
+      }
+      links.push({ to: '/admin/logs', children: <LGI>Application logs</LGI> })
+      if (isModuleEnabled('deployment')) {
+        links.push({ to: '/admin/servers', children: <LGI>Deployment servers</LGI> })
+      }
+    }
+    return links.map(link => (<LinkContainer key={link.to} {...link} />))
+  }
+
+  render () {
+    const { user } = this.props
+    const permissions = user && user.permissions
+    const isAdmin = permissions &&
+      (
+        permissions.isApplicationAdmin() ||
+        permissions.canAdministerAnOrganization()
+      )
+    const isApplicationAdmin = !!(permissions && permissions.isApplicationAdmin())
+    return (
+      <ManagerPage
+        ref='page'
+        title={this.messages('title')}>
+        <Grid>
+          <Row style={{ marginBottom: '18px' }}>
+            <Col xs={12}>
+              <h2>
+                <LinkContainer className='pull-right' to='/home'>
+                  <Button>
+                    Back to dashboard
+                  </Button>
+                </LinkContainer>
+                <Icon type='cog' /> {this.messages('title')}
+              </h2>
+            </Col>
+          </Row>
+          <Row>
+            {isAdmin
+              ? <div>
+                <Col xs={12} sm={3}>
+                  <Panel>
+                    <ListGroup fill>
+                      {this._getLinks(isApplicationAdmin)}
+                    </ListGroup>
+                  </Panel>
+                </Col>
+                <Col xs={12} sm={9}>
+                  {this._getMainContent(isApplicationAdmin)}
+                </Col>
+              </div>
+              : <div>
+                {user
+                  ? <p>{this.messages('noAccess')}</p>
+                  : <h1
+                    className='text-center'
+                    style={{ marginTop: '150px' }}>
+                    <Icon className='fa-spin fa-5x' type='refresh' />
+                  </h1>
+                }
+              </div>
+            }
+          </Row>
+        </Grid>
+      </ManagerPage>
+    )
+  }
+}
+
+const mapStateToProps = (state: AppState, ownProps: Props) => {
```

Comment:
`Props` is not the right props here. That is for the component that is connected. `ownProps` should be a `RouterProps` type.